examples/react-signals/src/App.css

```
@@ -0,0 +1,68 @@
+page {
```

Stylelint (17.14.0)

[error] Unknown type selector "page" (selector-type-no-unknown)

(selector-type-no-unknown)